.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: Gytheio CI
+
+on:
+  pull_request:
+    branches:
+      - master
+      - SP/**
+      - HF/**
+      - ACS-**
+      - feature/**
+  push:
+    branches:
+      - master
+      - SP/**
+      - HF/**
+      - ACS-**
+      - feature/**
+  schedule:
+    - cron: '0 5 * * 3'
+  workflow_dispatch:
+
+env:
+  MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+  MAVEN_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+  GIT_USERNAME: ${{ secrets.BOT_GITHUB_USERNAME }}
+  GIT_PASSWORD: ${{ secrets.BOT_GITHUB_TOKEN }}
+  AWS_REGION: eu-west-1
+
+jobs:
+  veracode:
+    name: "Source Clear Scan (SCA)"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.35.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.35.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/veracode@v1.35.0
+        with:
+          srcclr-api-token: ${{ secrets.SRCCLR_API_TOKEN }}
+
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Alfresco/alfresco-build-tools/.github/actions/get-build-info@v1.35.0
+      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.35.0
+      - name: "Remove SNAPSHOT artifacts"
+        run: find "${HOME}/.m2/repository/" -type d -name "*-SNAPSHOT*" | xargs -r -l rm -rf
+      - name: "Install ImageMagick to parse PDFs"
+        run: |
+          sudo apt-get update
+          sudo apt-get install -q -y  imagemagick ghostscript ffmpeg
+          sudo sed -i '/PDF/s/none/read|write/' /etc/ImageMagick-6/policy.xml
```

Comment:
It would be nice to add back the comment that was in the `init.sh` file, explaining what's the purpose of this line.